assets/src/edit-story/app/uploader/useUploader.js

```diff
@@ -108,40 +97,37 @@ function useUploader() {
         throw createError('SizeError', file.name, message);
       }
 
-      const additionalData = {
-        post: storyId,
-        media_source: 'editor',
-        ..._additionalData,
-      };
+      // The file is too large for the site anyway, abort.
```

Comment:
This chuck is repeated from the lines above. Let's remove it.

Reply:
Whoops, looks like I accidentally duplicated this 🤦